### DISCUSSION TITLE
Extended rem value not applied without space

### DISCUSSION THREAD

**Discussion by LgtmMurlock** [tailwindlabs/tailwindcss#8994] [Help] [answered]
Next.js project using version 12.1.6 and tailwindcss version ^3.0.24.

I have extended a few properties in tailwind.config.js, when extending the minHeight like so: 

```
minHeight: {
  20: '5 rem',
  'product-preview': '85 rem',
},
```

The rem is applied, but it is **not** applied when setting it like this (as the documentation suggests):
```
minHeight: {
  20: '5rem',
  'product-preview': '85rem',
},
```

**Comment by wongjn** [comment, marked as answer]
What's your exact Tailwind version? You specified a version constraint but not the exact version.

Anyway, try updating Tailwind to the latest version? I tried your config in [the playground](https://play.tailwindcss.com/ERAOzWLSaK?file=config) (which is running `v3.1.5` and it seems to be working correctly.

**Comment by LgtmMurlock** [comment]
Hi and thank you for getting back so quickly, the exact Tailwind version is 3.0.24. I didn't have the possibility to test more at the time of opening the issue but now after reinstalling everything it's working as expected again.

This behavior occurred on Windows and when I tried reproducing the behavior on my Macbook with the same version I couldn't, so perhaps there was an issue with caching or so on the Windows machine.
